tests/Cucumber/CompatibilityTest.php

```diff
@@ -46,7 +46,6 @@ class CompatibilityTest extends TestCase
         'extra_table_content.feature' => 'Table without right border triggers a ParserException',
         'incomplete_scenario_outline.feature' => 'Scenario and Scenario outline not yet synonyms',
         'padded_example.feature' => 'Scenario and Scenario outline not yet synonyms',
```

Comment:
what about those 2 that were skipped for the same reason ?

Reply:
They now fail for unrelated reasons.

* `incomplete_scenario_outline.feature` because we throw if an Outline has no Examples table, but cucumber/gherkin does not (tracked in #153 and noted in the diff above).
* `padded_example.feature` because we don't trim table padding (specifically &nbsp; I think) the same as cucumber/gherkin - no issue for this yet (I plan to create one), and it has BC implications.

I'll update the reason descriptions in the testcase